docs/REST-API/19-REST-API-IPs.md

```diff
@@ -1,7 +1,7 @@
 ---
 tags: [rest-api]
 ---
-> Notice: This safelist of IP addresses will take effect October 31st, 2023
+> Notice: This safelist of IP addresses will take effect December 18th, 2023
```

Comment:
nit: this statement is a little weird to me. Are these IPs not already in effect?

Reply:
It's a bit confusing, but what we're doing is _removing_ the legacy IPs and then giving 30 days notice that we're no longer using them, before finally releasing them. So effectively what we're saying is "as of December 18th, these are the _only_ IPs we'll be using for the REST API".